### PR TITLE
INT-2151 - CI Improvements:  Nexus Platform Plugin

### PR DESCRIPTION
**Pull request by kellyrob99**
- provide a profile for working supported environment and minor test changes to accommodate string syntax changes in certain error messages

### CODE REVIEW

File: src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy [hunk 1]
```diff
@@ -71,6 +72,7 @@ class IqPolicyEvaluatorIntegrationTest
     iqClientBuilder.withInstanceId(_) >> iqClientBuilder
     iqClient = Mock()
     iqClientBuilder.build() >> iqClient
+    println "Testing Jenkins version: ${Jenkins.getStoredVersion()}"
```
Comment (collinpeters) [inline src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy]: :+1:

File: pom.xml [hunk 2]
```diff
@@ -153,38 +158,30 @@
         <groupId>org.spockframework</groupId>
         <artifactId>spock-core</artifactId>
         <version>1.2-groovy-2.4</version>
-        <scope>test</scope>
```
Comment (whyjustin) [inline pom.xml]: Why is this moved to a compile scope?
